### ISSUE TITLE
Table: Table won't scroll up when using arrows on selected row

### ISSUE THREAD

**Issue by lukasmatta** [primefaces/primeng#11701]
### Describe the bug

Hello,

when using selectable table with virtual scrolling  enabled, one is unable to move up using arrows (when the row is focused). It stops on the first rendered row and table won't re-render. When moving down everything works as expected. I suppose it works downward, because there are more rows rendered under the last visible row.

### Environment

Angular: 13.3.11
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1303.8
@angular-devkit/build-angular   13.3.8
@angular-devkit/core            13.3.8
@angular-devkit/schematics      13.3.8
@angular/cdk                    13.3.9
@angular/cli                    13.3.8
@schematics/angular             13.3.8
rxjs                            7.5.5
typescript                      4.6.4

### Reproducer

https://stackblitz.com/edit/primeng-tablevirtualscroll-demo-g2u251?file=src/app/app.component.html

### Angular version

13.3.11

### PrimeNG version

13.4.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

16.13.1

### Steps to reproduce the behavior

Try to select any element in the middle of the table, and navigate upwards with arrow up.

### Expected behavior

I expect the virtual scroll to re-render elements as it works when going downwards.

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
